### DISCUSSION TITLE
'/payments/payment-operations' Return an empty 'marketplaceId'

### DISCUSSION THREAD

**Discussion by yao657355700** [allegro/allegro-api#8863] [Suggestions - rejected]
/payments/payment-operations
﻿
Why does this interface not return data of type "PAYOUT" after adding the parameter "marketplaceId"?
﻿
I tested it and found that when the parameter "marketplaceId" is not added, the returned "PAYOUT" data has an empty 'marketplaceId' value. 
Why is this happening?
![123](https://github.com/allegro/allegro-api/assets/87019541/14e4e55f-806c-4c93-a6a9-386a5ce9706c)

**Comment by MaciejFrackowiak** [comment]
This is not a suggestion - I have already answered in the thread https://github.com/allegro/allegro-api/issues/8868.
